modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RubyClientCodegen.java

```diff
@@ -141,6 +143,18 @@ public RubyClientCodegen() {
         cliOptions.add(new CliOption(CodegenConstants.HIDE_GENERATION_TIMESTAMP, CodegenConstants.HIDE_GENERATION_TIMESTAMP_DESC).
                 defaultValue(Boolean.TRUE.toString()));
 
+        cliOptions.add(CliOption.newBoolean(FARADAY, "Faraday (https://github.com/lostisland/faraday)"));
+        cliOptions.add(CliOption.newBoolean(TYPHOEUS, "Typhoeus (https://github.com/typhoeus/typhoeus)"));
+
+        supportedLibraries.put(FARADAY, "Faraday (https://github.com/lostisland/faraday)");
+        supportedLibraries.put(TYPHOEUS, "Typhoeus >= 1.0.1 (https://github.com/typhoeus/typhoeus)");
+
+        CliOption libraryOption = new CliOption(CodegenConstants.LIBRARY, "HTTP library template (sub-template) to use");
+        libraryOption.setEnum(supportedLibraries);
+        // set FARADAY as the default
+        libraryOption.setDefault(FARADAY);
```

Comment:
What about using Faraday as the default later (say 4.2.0) after we collect more feedback from the users?

Reply:
That would be great.